src/actions/receive-profile.js

```diff
@@ -572,9 +572,10 @@ export function waitingForProfileFromStore(): Action {
   };
 }
 
-export function waitingForProfileFromUrl(): Action {
+export function waitingForProfileFromUrl(profileUrl?: string): Action {
```

Comment:
I wish we could make that parameter non-optional but it looks like compare has some special handling :/